### DISCUSSION TITLE
Install for all users?

### DISCUSSION THREAD

**Discussion by sinoroc** [prefix-dev/pixi#1420] [Q&A] [answered]
Is it possible to install pixi (as root) for all users (on Linux)? If yes, how?

I have investigated a bit, but did not find anything conclusive.

**Comment by pavelzw** [comment, marked as answer]
Since this is a statically linked binary, you can just put it in `/usr/local/bin/pixi`.

```bash
curl -Ls \
    "https://github.com/prefix-dev/pixi/releases/download/v0.22.0/pixi-$(uname -m)-unknown-linux-musl" \
    -o /usr/local/bin/pixi
chmod +x /usr/local/bin/pixi
```

If you also want to add `~/.pixi/bin` to `$PATH` (for `pixi global install`), you could add something like this to `/etc/bash.bashrc`

```
export PATH="$HOME/.pixi/bin:$PATH"
```

This will adjust the `$PATH` in `bash`. You can do similar things in `zsh` and `fish` by modifying `/etc/zshenv` and `/etc/fish/config.fish`.

**Reply by sinoroc** [reply]
Thanks for that. Seems to work fine. Much more straightforward than I initially thought.

**Reply by pavelzw** [reply]
Actually you can also add the corresponding autocompletion line to the startup scripts as well for an even better experience: https://pixi.sh/dev/#autocompletion

**Reply by sinoroc** [reply]
Yes, adding to the following file seems to work fine (on Debian):

```
/usr/share/bash-completion/completions/pixi
```

Why the `eval` indirection, though? I feel like it makes starting `bash` a bit slower.

**Reply by ruben-arts** [reply]
The benefit of `eval` is that if you update `pixi` you'll automatically get the updated completions. But you're ofcourse free to output the result into a file and source that in the shell startup.
